Resources/Prototypes/_NF/Catalog/Fills/Items/weapon_cases_expedition.yml

```
@@ -219,6 +219,19 @@
     - id: MagazineBoxMagnum
       amount: 2
 
+- type: entity #made By VividPups added in with PR 1770
```

Repeater doesn't have this comment.  Seems odd to me.  At least fix the PR number.
```suggestion
- type: entity #made By VividPups added in with PR 1417
```